create_aio_app/template/{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/utils.py

```diff
@@ -50,6 +50,10 @@ def get_config(argv: Any = None) -> Any:
     return commandline.config_from_options(options, CONFIG_TRAFARET)
 
 
-def init_config(app: web.Application, *, config: list = None) -> None:
+def init_config(
+        app: web.Application,
+        *,
+        config: Optional[Dict[str, Dict[str, str]]] = None
+) -> None:
     app['config'] = \
         get_config(config or ['-c', DEFAULT_CONFIG_PATH.as_posix()])
```

Comment:
@svalee `config or [...]`
`dict` or `list`? I don't think so.

Reply:
@webknjaz yeah, my bad, sorry

Reply:
No worries :) You're doing great!